scheduler.go

```diff
@@ -21,23 +22,29 @@ var (
 	State     ecs_state.StateOps // State instance of ecs_state
 	Cluster   string             // Cluster the name of the ECS cluster
 	AWSRegion string             // AWSRegion AWS Region of the cluster
+	Interval  int                // Interval the time interval of ecs state refresh
 )
 
 // Run starts the scheduler
 func Run(port string, isDebug bool) error {
 	Sess, _ = session.NewSession(&aws.Config{Region: aws.String(AWSRegion)})
 	Client = ecs.New(Sess)
 	State = ecs_state.Initialize(Cluster, Client)
-	State.RefreshClusterState()
-	State.RefreshContainerInstanceState()
-	State.RefreshTaskState()
+	job.Every(Interval).Minutes().Run(Refresh)
```

Comment:
Don't we want to refresh when we start? Otherwise the cluster state is just empty?